project/build.gradle

```diff
@@ -7,9 +7,13 @@ plugins {
 group = 'com.cs4337'
 version = '0.0.1-SNAPSHOT'
 
+bootJar {
+	duplicatesStrategy = DuplicatesStrategy.EXCLUDE
+}
+
 java {
 	toolchain {
-		languageVersion = JavaLanguageVersion.of(23)
```

Comment:
Probably best to keep the language version at 23 unless it breaks a dependency, since the build scripts in ci/cd and docker already consider java 23